CHANGELOG.md

```diff
@@ -7,14 +7,15 @@
 - [#443](https://github.com/influxdata/clockface/pull/443): Enable `TreeNavHeader` to utilize the `ComponentColor` type
 - [#442](https://github.com/influxdata/clockface/pull/442): Update design of `InfluxDBCloudLogo` to conform to brand guidelines
 - [#441](https://github.com/influxdata/clockface/pull/441): Fix rendering of Rubik font by using `woff2` format instead of `ttf`
+- [#449](https://github.com/influxdata/clockface/pull/449): Added an onClick prop to the Heading component to allow headings to be clicked
```

Comment:
Super nitpicky, but can you make a header above this one for the current version and move your entry to beneath that header?

`#### 2.0.2 (Unreleased)`

And then remove the `(Unreleased)` from the 2.0.1 header

Reply:
Totally NBD